pkg/controller/azvolume.go

```diff
@@ -155,7 +155,10 @@ func (r *ReconcileAzVolume) triggerCreate(ctx context.Context, azVolume *diskv1b
 		defer cloudCancel()
 
 		var updateFunc func(interface{}) error
+		// UpdateCRIWithRetry should be called on a context w/o timeout when called in a separate goroutine as it is not going to be retriggered and leave the CRI in unrecoverable transient state instead.
+		updateCtx := context.Background()
```

Comment:
Why move this here when it was near where it was being used before? It's not being used elsewhere in the PR.